feature/home/src/main/java/com/yapp/feature/home/setting/SettingViewModel.kt

```
@@ -40,6 +48,26 @@ class SettingViewModel @Inject constructor() : ViewModel() {
+
+            SettingIntent.DismissDeleteAccountDialog,
+            SettingIntent.ClickDeleteAccountDialogActionButton -> {
+                reduce { copy(showDeleteAccountDialog = false) }
+            }
+
+            SettingIntent.DismissLogoutDialog,
+            SettingIntent.ClickLogoutDialogActionButton -> {
+                reduce { copy(showLogoutDialog = false) }
+            }
+
+            SettingIntent.ClickDeleteAccountDialogRecommendActionButton -> TODO()
```

detekt (1.23.7)

[warning] The NotImplementedDeclaration should only be used when a method stub is necessary. This defers the development of the functionality of this function. Hence, the `NotImplementedDeclaration` should only serve as a temporary declaration. Before releasing, this type of declaration should be removed.

(detekt.exceptions.NotImplementedDeclaration)